src/parser/core/modules/BuffWindow.tsx

```diff
@@ -288,7 +288,7 @@ export abstract class BuffWindowModule extends Module {
 	}
 
 	private getBuffWindowExpectedTrackedActions(buffWindow: BuffWindowState, action: BuffWindowTrackedAction): number {
-		return this.getBaselineExpectedTrackedAction(buffWindow, action) + this.changeExpectedTrackedActionClassLogic(buffWindow, action)
+		return this.getBaselineExpectedTrackedAction(buffWindow, action) + this.changeExpectedTrackedActionClassLogic(buffWindow, action) - this.reduceExpectedGCDsEndOfFight(buffWindow)
```

Comment:
This should be a separate function rather than reusing the GCD reducer.  By default this would automatically reduce any expected single use per window action to 0 if the window was rushed at all, and this doesn't provide a way for implementing classes to determine how to handle rushing per-action.